specification/iotoperations/IoTOperations.Management/models/broker/brokers.tsp

```diff
@@ -56,18 +55,215 @@ model BrokerProperties {
   @doc("Memory profile of Broker.")
   memoryProfile?: BrokerMemoryProfile = BrokerMemoryProfile.Medium;
 
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  @doc("The persistence settings of the Broker.")
+  @added(Versions.`2025-07-01-preview`)
+  persistence?: BrokerPersistence;
+
   @visibility(Lifecycle.Read)
   @doc("The status of the last operation.")
   provisioningState?: ProvisioningState;
 }
 
+@doc("""
+  Disk persistence configuration.
+  
+  When persistence is enabled, certain items (non-performance-critical data) selected for persistence will reside only on disk. Below are the affected items: 
+  
+  - Retained messages will be stored on disk only. 
+  - WILL messages will be stored on disk only. 
+  - DSS key/value pairs will be stored on disk only, except for performance-critical items like timed locks, which remain in both disk and memory for improved performance.
+  
+  Optional. Everything is in-memory if not set. 
+  Note: if configured, all MQTT session states are written to disk.
+  """)
+@added(Versions.`2025-07-01-preview`)
+model BrokerPersistence {
+  @doc("""
+    Client sets the specified user property key/value in the CONNECT/SUBSCRIBE/PUBLISH 
+    Optional, default shown If customer specify a user property, it will work to enable persistence dynamically, in addition to the default user property "aio-persistence"
```

Comment:
nit: 'If' -> 'if', also maybe just "Optionally, if the customer specifies a configurable user property, it will work to enable persistence dynamically. The default user property key is 'aio-persistence' and value 'true'.".